### ISSUE TITLE
Blazor WASM with Static Web App Random net::ERR_HTTP2_PROTOCOL_ERROR

### ISSUE THREAD

**Issue by asperpint** [dotnet/aspnetcore#42256]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Creating separate issue as per @javiercn .

We recently updated to .NET 6.0.300. Our production apps deployed on Azure Static Apps and Standalone Blazor WASM started throwing random net::ERR_HTTP2_PROTOCOL_ERROR.

https://consumer.perpetualintelligence.com/ 

The error is random and hitting F5 makes it go away. The typical sign of the error is it take a long time to load and then times out with Http2 error. This is a work network in US so there was no network interruption. I also have Azure portal open so that is also working fine.

![image](https://user-images.githubusercontent.com/106403372/174276408-6719042f-7bfe-4806-9114-17e5b292f85c.png)

After Error F5 loads it correctly.
![image](https://user-images.githubusercontent.com/106403372/174276687-0c349552-7e18-4884-8eb8-b94ed3b26833.png)

### .NET Version

6.0.300

### Anything else?

VS2022

**Comment by javiercn** [comment]
@asperpint can you tell us what version did you upgrade from?

Are you able to reproduce this issue in another browser like Firefox or Safari? (I assume this is Edge/Chrome)

**Comment by javiercn** [comment]
@anthonychu Is this something you can help with?

**Comment by anthonychu** [comment]
Adding @mkarmark to help investigate.

I haven't been able to repro this by going to the site. @asperpint please let us know if you can reproduce this in a browser other than Chrome or on another machine.

**Comment by anthonychu** [comment]
Just saw this other issue. Looks like it's happening on other apps hosting on Static Web Apps, and also in other browsers. https://github.com/dotnet/aspnetcore/issues/42203

**Comment by mishapos** [comment]
@asperpint Hi, looking into this from the Static Web Apps side as well and have not been able to reproduce this behavior with your Static Web App from my browsers. If you are still experiencing this issue, could you please open a support ticket with Azure and route it to Azure Static Web Apps, so we can get the network trace and see what's going on? Thanks!

**Comment by asperpint** [comment]
@mishapos Thanks. I am consistently able to reproduce in incognito. However, one difference was that the Static app is deployed in West US, and I have been testing and getting this issue in other data centers, e.g., in South Asia.

I plan to deploy a test environment for the static web app, use the Edge optimization and test its VMs across multiple geographic data centers. I will let you know the results soon.

**Comment by mkArtakMSFT** [comment]
Closing as there is no action pending on our team and Static Web Apps team is unable to reproduce it.
Per @mishapos 's recommendation, please open a support ticket with Azure so that it gets further attention.
